alembic/env.py

```diff
@@ -0,0 +1,82 @@
+from logging.config import fileConfig
+
+from sqlalchemy import create_engine, pool, engine_from_config
+from alembic import context
+from app.models import Base
```

Comment:
Imports should be grouped and separated by a blank line according to standard library, external libraries, and custom imports. This practice enhances both code readability and maintainability. 
Read more about topic [here](https://peps.python.org/pep-0008/#imports)